components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift

```diff
@@ -83,10 +83,14 @@ class ButtonsSwiftAndStoryboardController: UIViewController {
 
     let floatingPlusShapeLayer = ButtonsTypicalUseSupplemental.createPlusShapeLayer(floatingButton)
     floatingButton.layer.addSublayer(floatingPlusShapeLayer)
+    floatingButton.applySecondaryTheme(withScheme: containerScheme)
+    floatingButton.accessibilityLabel = "Programmatic FAB"
```

Comment:
Consider using "floating action button" here to reduce reliance on acronyms.

Reply:
Done